src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -146,11 +149,11 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods, sites }) => {
             let color = 'green';
 
             // Use this for coloring reserved/completed blocks a different color
-            if (feature.getProperty('ID') % 10 === 0) {
+            if (feature.getProperty('block_id') % 10 === 0) {
```

Comment:
💭 I wonder if we should have a `BlockProperties` enum (for neighborhoods and sites too) with all of the property names. Would make it easier to keep these consistent